pyproject.toml

```
@@ -39,6 +39,7 @@ dependencies = [
     "ipypopout>=1.4.0",
     "astroquery",
     "requests",
+    "pyvo>=1.5.3",
```

Can we add some sort of comment so that we remember that this is pulled in by astropy and can be removed once we no longer need a newer version?

```suggestion
    "pyvo>=1.5.3",  # NOTE: can be unpinned if/when astroquery requires the same version
```

---

Right! Sorry. Added in in https://github.com/spacetelescope/jdaviz/pull/2872/commits/42bdb01eb50a118ac505143e7c67e990c3bb4a36